pkg/cmd/codespace/codespace_selector_test.go

```diff
@@ -48,68 +48,94 @@ func TestSelectNameWithCodespaceName(t *testing.T) {
 
 func TestFetchCodespaces(t *testing.T) {
 	var (
-		repoA1 = &api.Codespace{Name: "1", Repository: api.Repository{FullName: "mock/A"}}
-		repoA2 = &api.Codespace{Name: "2", Repository: api.Repository{FullName: "mock/A"}}
-
-		repoB1 = &api.Codespace{Name: "1", Repository: api.Repository{FullName: "mock/B"}}
+		octocatOwner = api.RepositoryOwner{Login: "octocat"}
+		cliOwner     = api.RepositoryOwner{Login: "cli"}
+		repoA1       = &api.Codespace{
+			Name:       "1",
+			Repository: api.Repository{FullName: "mock/A", Owner: octocatOwner},
```

Comment:
For consistency, can we make the owner names align with the repo `FullName`s? For example, `mock/A` should become `octocat/A`. That probably means we should add a few more codespaces to this list to cover all the test cases as well

Reply:
Added one more test seed, let me know if I am missing any specific cases!